types/lib/utils/gestures.d.ts

```diff
@@ -42,15 +42,15 @@ declare namespace Polymer {
      *
      * @returns Returns true if a gesture event listener was added.
      */
-    function addListener(node: Node, evType: string, handler: Function): boolean;
+    function addListener(node: Node, evType: string, handler: (p0: Event|null) => void): boolean;
```

Comment:
i need help here. `handler` should have an optional parameter, rather than just null.. `(p0?: Event|null)` (assuming strict null checks was on). my annotation was `function(?Event=):void` before, but that seems to produce `(p0: any)`...

Reply:
The converter was missing the ability to parse optional and rest parameters in type expressions. so it was reverting to `any`. Will be fixed in https://github.com/Polymer/gen-typescript-declarations/pull/51. When that's merged, I'll do a patch release and you should be able to use the correct syntax.

Reply:
This PR is released in `0.3.4` so bump the dependency and you should be able to get the right output for `function(?Event=):void`.